Packs/FeedMandiant/Integrations/FeedMandiant/FeedMandiant.yml

```diff
@@ -0,0 +1,212 @@
+  name: xappname
+  required: true
+  type: 0
+- display: Public Key
+  displaypassword: Secret Key
+  name: auth
+  required: true
+  type: 9
+- defaultvalue: '50'
+  display: Maximum number of indicators per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- additionalinfo: 'The indicators'' type to fetch. Indicator type might include the
+    following: Domains, IPs, Files and URLs.'
+  defaultvalue: Malware,Actors,Indicators
+  display: Indicator Type
+  name: type
+  options:
+  - Malware
+  - Actors
+  - Indicators
+  required: false
+  type: 16
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
+- additionalinfo: Retrieve additional information for each indicator. Please note
+    that this requires additional API calls.
+  defaultvalue: 'false'
+  display: Retrieve Indicator Metadata
+  name: indicatorMetadata
+  required: false
+  type: 8
+- additionalinfo: Please note that this requires additional API calls.
+  defaultvalue: 'false'
+  display: Create Relationships.
+  name: indicatorRelationships
+  required: false
+  type: 8
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+  display: Indicator Reputation
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- additionalinfo: Reliability of the source providing the intelligence data
+  defaultvalue: A - Completely reliable
+  display: Source Reliability
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: ''
+  name: feedExpirationPolicy
+  options:
+    - never
+    - interval
+    - indicatorType
+    - suddenDeath
+  required: false
+  type: 17
+- display: ''
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- defaultvalue: '240'
+  display: Feed Fetch Interval
+  name: feedFetchInterval
+  required: false
+  type: 19
+- defaultvalue: Low
+  display: Minimum severity of alerts to fetch
+  name: min_severity
+  options:
+  - Low
+  - Medium
+  - High
+  - Critical
+  required: true
+  type: 15
+- additionalinfo: API calls timeout.
+  defaultvalue: '60'
+  display: Timeout
+  name: timeout
+  required: false
```

Comment:
When running fetch indicator commands with on all the available indicator, the default 10 seconds timeout was not enough.
On my local machine increasing the timeout to 30 solved the issue. 60 seconds seems like a reasonable time to prevent similar issues for the average user.

Reply:
This is for one API call which to me sounds a lot (you can have thousands of indicators right?), how does it affect the time performance of the fetch process if each API request takes 30 secs, for each indicator you might perform 4 API calls, and there might be thousands of indicators?